dev-tools/cosmos-records/sync-orders.ts

```diff
@@ -23,4 +23,7 @@ export function syncOrders() {
     .catch((error) => console.log('Unable to sync orders. Reason:', error.message));
 }
 
+// if (require.main === module) {
+//   syncOrders();
+// }
 syncOrders();
```

Comment:
Was this a mistake?  Did we mean to uncomment the module check or delete it?

Reply:
We need to figure out how to write this guard is the right answer.

Reply:
Deleted the commented code.